nearcore/src/state_sync.rs

```diff
@@ -492,51 +466,34 @@ fn obtain_and_store_state_part(
     Ok(state_part)
 }
 
-/// Gets basic information about the epoch to be dumped.
-fn start_dumping(
-    epoch_id: EpochId,
-    sync_hash: CryptoHash,
-    shard_id: ShardId,
+fn cares_about_shard(
```

Comment:
with start_dumping() gone, I think dumping wouldn't be impacted, only issue is the `STATE_SYNC_DUMP_NUM_PARTS_DUMPED`. If we still want `STATE_SYNC_DUMP_NUM_PARTS_DUMPED` to represent the parts dumped by this node, then we need to know when to start counting, i.e. initialize it to 0, and then we increment by 1 every time a part is dumped on this node. And we would be able to observe how each node's dumping has been progressing with this setup.

Reply:
I'd rather have `STATE_SYNC_DUMP_NUM_PARTS_DUMPED` represent the aggregated information about the epoch in general. Averaging this over multiple nodes will give a conservative  approximation of the epoch state dumping progress.